src/components/Question/Question.tsx

```diff
@@ -13,43 +13,59 @@ interface Props {
 export const Question: React.FC<Props> = ({ updateAllAnswers }) => {
 
   const questionContext = useContext(QuestionContext);
-  const questionKeys = Object.keys(questionContext )
+  const questionKeys = Object.keys(questionContext)
   const [answerInput, updateAnswer] = useState<any>({});
   const [index, setIndex] = useState<number>(0);
+  const[isSelected, setIsSelected] = useState<boolean>(false)
   const [errorMessage, setErrorMessage] = useState<string>("");
+  const[rentSelected, setRentSelected] = useState<boolean>(false);
+  const[principleSelected, setPrincipleSelected] = useState<boolean>(false)
   const inputRef = useRef<HTMLInputElement>(null);
-  
+
   const organizeData = () => {
     questionKeys.sort((a:any, b:any) => questionContext[a]['03_attributes']
     .A_order > questionContext[b]['03_attributes']
     .A_order ? 1 : -1)
   }
 
   organizeData()
-  
+
   let currentQuestion = questionContext[questionKeys[index]];
 
   useEffect(() => {
     inputRef?.current?.focus();
   },[index])
-  
-  
+
+
   useEffect(() => {
     inputRef?.current?.focus();
   },[])
-  
+
+  const displayInput = (e: any) => {
+    setIsSelected(true)
+    if(e.target.id === 'principle') {
+      setPrincipleSelected(true);
+      setRentSelected(false);
+      setIndex(7);
+      updateAnswer({ ...answerInput, [questionKeys[8]]: '0'});
+    } else if (e.target.id === 'rent') {
+      setPrincipleSelected(false);
+      setRentSelected(true);
+      setIndex(8);
+      updateAnswer({ ...answerInput, [questionKeys[7]]: '0'});
+    }
+  }
+
```

Comment:
This was a great addition to handle the question's logic. Thanks for your hard work!!